cdc/sinkv2/tablesink/table_sink.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tablesink
+
+import "github.com/pingcap/tiflow/cdc/model"
+
+// TableSink is the interface for table sink.
+// It is used to sink data in table units.
+type TableSink interface {
+	// AppendRowChangedEvents appends row changed events to the table sink.
+	// Usually, it is used to cache the row changed events into table sink.
+	AppendRowChangedEvents(rows ...*model.RowChangedEvent) error
+	// UpdateResolvedTs writes the buffered row changed events to the TxnEventSink/RowEventSink.
+	// Note: This is an asynchronous method.
+	UpdateResolvedTs(resolvedTs model.ResolvedTs) error
+	// GetCheckpointTs returns the current checkpoint ts of table sink.
+	// Usually, it requires some computational work.
+	// For example, calculating the current progress from the minimum heap.
+	GetCheckpointTs() (model.ResolvedTs, error)
+	// Close closes the table sink.
+	// 1) set the table status to stopped.
+	// 2) clean up the table sink buffer.
```

Comment:
It's better not to focus on implementation details if you're writing comments on the interface.

Reply:
Removed.